EventFilter/GEMRawToDigi/python/gemPacker_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
If the full fillDescriptions is provided, the cfi file will be auto-generated, so this could be removed. (same for others below)